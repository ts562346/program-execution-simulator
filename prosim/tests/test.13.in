test1 12
BLOCK 1
LOOP 4
  DOOP 2
  BLOCK 3
END
DOOP 5
LOOP 3
  BLOCK 3
  DOOP 4
END
BLOCK 7
HALT

