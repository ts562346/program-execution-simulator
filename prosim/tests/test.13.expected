00000: BLOCK
00001: DOOP
00003: BLOCK
00006: DOOP
00008: BLOCK
00011: DOOP
00013: BLOCK
00016: DOOP
00018: BLOCK
00021: DOOP
00026: BLOCK
00029: DOOP
00033: BLOCK
00036: DOOP
00040: BLOCK
00043: DOOP
00047: BLOCK
00054: HALT

DOOP count : 8
DOOP time  : 25
BLOCK count: 9
BLOCK time : 29
